client/src/pages/CataloguePage/CataloguePage.js

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { Stack, Container, Box, Typography } from '@mui/material';
+import CatalogTourCard from '../../components/CatalogTourCard/CatalogTourCard';
```

Comment:
I suggest adding `CatalogTourCard` to index.js in components folder and make import like this:
```suggestion
import { CatalogTourCard } from '../../components';
```

Reply:
done